asmcli/lib/parse.sh

```diff
@@ -70,6 +70,7 @@ parse_args() {
         OPTIONAL_OVERLAY="${2},${OPTIONAL_OVERLAY}"
         context_set-option "OPTIONAL_OVERLAY" "${OPTIONAL_OVERLAY}"
         if [[ "${2}" == "hub-meshca" ]]; then
+          info "Fleet workload identity pool is used as default for Mesh CA. No need to specify hub-meshca option."
           context_set-option "USE_HUB_WIP" 1
```

Comment:
If there's no need to specify it, then we shouldn't need to set the USE_HUB_WIP context, right?

Reply:
USE_HUB_WIP is set to 1 because the Hub configuration flow (reading Hub resources and parse Hub meta such as IdP) is controlled by this flag. Maybe as Hub is default,  USE_HUB_WIP can have the default value 1 and will only be set to 0 when user does not specify enable_all or enable_registration, and the cluster has not been registered into Hub before the installation. Basically when `! is_managed && [[ "${CA}" == "mesh_ca" ]] && ( is_gcp && ! can_register_cluster && ! is_cluster_registered )`. Do you think this is better?